packages/manager/apps/hub/src/index.js

```diff
@@ -8,7 +8,6 @@ import 'ovh-ui-angular';
 import ovhManagerCore from '@ovh-ux/manager-core';
 import ovhManagerHub from '@ovh-ux/manager-hub';
 import ovhManagerNavbar from '@ovh-ux/manager-navbar';
-import ovhManagerHub from '@ovh-ux/manager-hub';
```

Comment:
How could this be missed by eslint ? 🤔

Reply:
@FredericEspiau An editor linter is able to report you this kind of issue.

Also ESLint is able to prevent you to commit this but we probably rely on an issue when resolving conflicts.